src/main/java/uk/gov/hmcts/juror/api/moj/controller/request/JurorManagementRequestDto.java

```diff
@@ -2,11 +2,14 @@
 
 import com.fasterxml.jackson.annotation.JsonProperty;
 import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.validation.constraints.NotEmpty;
+import jakarta.validation.constraints.NotNull;
 import jakarta.validation.constraints.Size;
 import lombok.AllArgsConstructor;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
 import lombok.Setter;
+import org.hibernate.validator.constraints.Length;
```

Comment:
might want to check imports, cannot see usage of these additional imports

Reply:
Fixed